src/mon/MonmapMonitor.cc

```diff
@@ -200,6 +200,11 @@ bool MonmapMonitor::preprocess_command(MonOpRequestRef op)
       p = new MonMap;
       p->decode(bl);
     }
+    else {
```

Comment:
maybe we can just kill the check at https://github.com/ceph/ceph/pull/9650/files#diff-48d6f7c4cfa3781fd3f8ed5acd51b886R191.